src/seedsigner/models/settings_definition.py

```diff
@@ -699,6 +689,14 @@ class SettingsDefinition:
                       visibility=SettingsConstants.VISIBILITY__ADVANCED,
                       default_value=SettingsConstants.OPTION__ENABLED),
 
+        SettingsEntry(category=SettingsConstants.CATEGORY__SYSTEM,
+                      attr_name=SettingsConstants.SETTING__WORDLIST_LANGUAGE,
+                      abbreviated_name="wordlist_lang",
+                      display_name=_mft("Mnemonic language"),
+                      type=SettingsConstants.TYPE__SELECT_1,
+                      selection_options=SettingsConstants.ALL_WORDLIST_LANGUAGES,
+                      visibility=SettingsConstants.VISIBILITY__ADVANCED,
+                      default_value=SettingsConstants.WORDLIST_LANGUAGE__EN),
```

Comment:
This should probably be relocated to be just after "BIP-39 passphrase" to keep the related options together.